tensorflow/python/client/session_clusterspec_prop_test.py

```diff
@@ -106,7 +106,7 @@ def testClusterSpecPropagationWorker1Placement(self):
     with ops.Graph().as_default() as g, ops.device('/job:worker/task:0'):
       const = constant_op.constant(17)
     sess = session.Session(server1.target, config=config, graph=g)
-    output = self.evaluate(const)
+    output = sess.run(const)
```

Comment:
Quick question: why do you switch from `self.evaluate` to `sess.run`?

Reply:
Because, the created Session is not used as a context, so the `ops.get_default_session()` behind the `self.evaluate(...)` will return None and a test Session will be created without the server info.
If `self.evaluate(...)` is preferred, we can use the following instead:
``` python
with session.Session(server1.target, config=config, graph=g) as sess:
  output = self.evaluate(const)
  self.assertEqual(17, output)
```

Reply:
Done with a new [commit](https://github.com/tensorflow/tensorflow/pull/24466/commits/88833ed2e8eaaef44e8c7a5d9a3d9b81c5abb497) which explicitly set the session context.